charts/redhat/redhat/postgresql-persistent/0.0.1/values.yaml

```diff
@@ -0,0 +1,22 @@
+database_service_name: postgresql-testing
+memory_limit: 512Mi
+namespace: postgresql-persistent-testing
+volume_capacity: 1Gi
+config:
+  postgresql_database: testdb
+  postgresql_password: testp
+  postgresql_user: testu
+  port: 5432
+image:
+  tag:
+    - "latest"
+    - "13-el9"
+    - "13-el8"
```

Comment:
This duplicate tag is probably the issue with the lint

Reply:
Thanks stupid typo.